streams/streams-scala/src/main/scala/org/apache/kafka/streams/scala/ImplicitConversions.scala

```diff
@@ -19,19 +19,14 @@
  */
 package org.apache.kafka.streams.scala
 
-import org.apache.kafka.streams.kstream.{KStream => KStreamJ,
-  KTable => KTableJ,
-  KGroupedStream => KGroupedStreamJ,
-  SessionWindowedKStream => SessionWindowedKStreamJ,
-  TimeWindowedKStream => TimeWindowedKStreamJ,
-  KGroupedTable => KGroupedTableJ, _}
-
+import org.apache.kafka.streams.kstream.{KGroupedStream => KGroupedStreamJ, KGroupedTable => KGroupedTableJ, KStream => KStreamJ, KTable => KTableJ, SessionWindowedKStream => SessionWindowedKStreamJ, TimeWindowedKStream => TimeWindowedKStreamJ, _}
```

Comment:
Is it intentional to put them in a single line? Multi-lines are more human readable.

Reply:
No, this change has been introduced by my IDE and doesn't make sense. I will revert it.